features/custom_codecs/README.md

```diff
@@ -374,6 +374,45 @@ Foo<Bar> foo = row.get(0, new TypeToken<Foo<Bar>>(){})
 Foo<Bar> foo = row.get(0, Foo.class);
 ```
 
+### Support for subtype polymorphism
```

Comment:
I really like the explanation here.  It is very concise and helps me understand how this works.